WalletWasabi.Fluent/Global.cs

```diff
@@ -98,6 +100,7 @@ public Global(string dataDir, Config config, UiConfig uiConfig, WalletManager wa
 			Synchronizer = new WasabiSynchronizer(BitcoinStore, HttpClientFactory);
 			LegalChecker = new(DataDir);
 			TransactionBroadcaster = new TransactionBroadcaster(Network, BitcoinStore, HttpClientFactory, WalletManager);
+			TorStatusChecker = new TorStatusChecker(TimeSpan.FromHours(6), HttpClientFactory.NewHttpClient(Mode.DefaultCircuit), new XmlIssueListParser());
```

Comment:
@kiminuo Please, notice how I injected the `HttpClient`. I need to know if this is the correct way to do it.

Reply:
I've added the 3 commits, except for "Remove measure code from `Global` as it reports < 200ms times"